enterprise/static/enterprise/grant_data_sharing_permissions.css

```diff
@@ -200,6 +198,13 @@ h2#modal-header-text {
     font-size: 18px;
 }
 
+.modal-content .scrollable {
+    max-height: 90vh;
```

Comment:
nit: you could set it to `calc(90vh - 30px)` to account for the height of the top bar with the close (X) button. That way there will always be some vertical space between the window and the popup (currently if you make the height of the window small enough, the popup fills the entire height).